src/us/kbase/genbank/test/TestCommandLine.java

```diff
@@ -25,7 +25,7 @@
  * Date: 10/29/14
  * Time: 12:17 AM
  */
-public class CommandLineTest {
+public class TestCommandLine {
```

Comment:
I renamed @realmarcin's class so it wouldn't look like a class with JUnit unit tests.  Here
and elsewhere in KBase, those classes are in a test/ directory and also named *Test.java.
This seems to be the most popular convention:
http://stackoverflow.com/questions/3146821/naming-convention-junit-suffix-or-prefix-test
Besides FormatTest, which I added, I don't know what the other test classes do, or even if they contain any code that still tests anything useful.  But none of the others contain unit tests, and most seem to follow a Test*.java naming convention, so I renamed this one for consistency.